power/battery.go

```diff
@@ -1,3 +1,5 @@
+// +build !linux
```

Comment:
So I understand: this tells the compiler not to compile this module on Linux....